arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java

```diff
@@ -51,19 +51,23 @@ public String toString() {
 
   @Override
   public void enter() {
-    this.lists.push(new CountedList<>());
+    this.lists.addLast(new CountedList<>());
   }
 
   public void enter(int initialCapacity) {
-    this.lists.push(new CountedList<>(initialCapacity));
+    this.lists.addLast(new CountedList<>(initialCapacity));
+  }
+
+  public E getLast() {
+    return this.lists.getLast().getLast();
```

Comment:
What is the explanation for the double `getLast` ?

Reply:
BTW would `getLatest` be more descriptive and accurate than `getLast` ?

Reply:
`getLast` is the default Java method name for such accessors.